File: P1S2/src/GrupoPracticas.java
public class GrupoPracticas extends Grupo {
    private AulaPracticas aula;

    public GrupoPracticas(String nombre) {
        super(nombre);
        System.out.println("prácticas.");
    }

    @Override
    void asignarAula(Aula aula) {
        this.aula = (AulaPracticas) aula;
    }

    @Override
    public String toString() {
        return "GrupoPracticas{" +
                "nombre='" + nombre + '\'' +
                ", aula=" + aula +
                '}';
    }
}

File: P1S2_Ruby/grupo_practicas.rb
require_relative 'grupo'

class GrupoPracticas < Grupo
  def initialize(nombre)
    super
    puts "Creado grupo de prácticas #{@nombre}"
  end

  def to_s
    "GrupoPracticas{nombre='#{@nombre}', aula=#{@aula&.nombre}}"
  end
end
